src/backend/services/users-api/src/databases/UserSchema.py

```diff
@@ -0,0 +1,28 @@
+userSchema = {
+        "$jsonSchema": {
+            "bsonType": "object",
+            "required": ["first_name", "last_name", "username", "password", "public_key"],
```

Comment:
`username` -> `umnetId` change in all DB stuff too please

Reply:
I think `username` makes more sense from the backend perspective. `umnetid` will be the username sure, but we don't need our microservices to know that for abstraction purposes